models.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False, unique=True)
+    bio = Column(String(511), nullable=False)
+
+
+class DBBook(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), nullable=False)
+    summary = Column(String(511), nullable=False)
+    publication_date = Column(Date, nullable=False)
+    author_id = Column(Integer, ForeignKey("author.id"))
+
+    author = relationship(DBAuthor)
```

Comment:
Consider specifying the `back_populates` parameter in the `relationship` definition to establish a bidirectional relationship between `DBAuthor` and `DBBook`. This can help with navigating relationships in both directions and is useful for ORM operations.